Utter/ContentView.swift

```diff
@@ -13,7 +13,10 @@ struct ContentView: View {
             Image(systemName: "globe")
                 .imageScale(.large)
                 .foregroundStyle(.tint)
-            Text("Utterly")
+			Text("Utterly")
+				.font(.title3)
```

Comment:
```suggestion
				.font(.title2)
```
a big bigger - can I write here?